cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -118,6 +121,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
         help = "Do not scan excluded projects or packages. Works only with the '--ort-file' parameter."
     ).flag()
 
+    private val archiveAll by option(
```

Comment:
Let's move this commit to a separate PR as I believe it will trigger longer discussions.